### ISSUE TITLE
Issues while using the versions v4.2.1 and v3.1.1

### ISSUE THREAD

**Issue by Gokulnath-Venugopal** [hasura/smooth-checkout-buildkite-plugin#42]
I need to use the latest version due to the change in GITHUB.COM RSA keys. Kindly release the new version with the below issues fixed pls.

1. Error while using the version 3.1.1

```
🧹 Cleaning up workspace | 1m 29s
-- | --
  | Removing directory: /var/lib/buildkite-agent/buildkite-checkouts/018725ce-3365-48a3-bd56-cab71be669c0/018725d2-6c21-4972-bbae-5c137129b7cd/
  |  
  | We trust you have received the usual lecture from the local System
  | Administrator. It usually boils down to these three things:
  |  
  | #1) Respect the privacy of others.
  | #2) Think before you type.
  | #3) With great power comes great responsibility.
  |  
  | [sudo] password for buildkite-agent: # Received cancellation signal, interrupting
  |  
  | 🚨 Error: Error tearing down bootstrap: The plugin smooth-checkout pre-exit hook exited with status -1
```

2. Error while using the version 4.2.1

```
_📂 Setting up workspace | 0s
-- | --
  | Creating directory: /var/lib/buildkite-agent/builds/buildkite-tellus-microservice-1-i-001cff39f0882222a-1/organic-rssssse-1/tellus-microservice
  | Setup completed successfully.
  | 🔑 Setting up git and ssh | 0s
  | Checking out repository |  
  | 0
  | :::: local clone location = '.'
  | :::: checkout ref = 'master'
  | :::: git remote url = 'git@bitbucket.org:orgaxxxxxxxx/portal-dddd.git'
  | :::: ssh key path = ''
  | :::: clone flags = ''
  | Git repository already exists.
  | So, doing a git clean and setting the remote URL
  |  
  | We trust you have received the usual lecture from the local System
  | Administrator. It usually boils down to these three things:
  |  
  | #1) Respect the privacy of others.
  | #2) Think before you type.
  | #3) With great power comes great responsibility.
  |  
  | [sudo] passw_
```

**Comment by scriptnull** [comment]
Hi, could we know the version that you were using prior to v4.2.1 and v3.1.1?

**Comment by Gokulnath-Venugopal** [comment]
I was using v4.0.0 and it used to be stable..

**Comment by Gokulnath-Venugopal** [comment]
Any updates on this?

**Comment by scriptnull** [comment]
Yeah! I have prepared two tags for you to try out.

- [v4.0.3](https://github.com/hasura/smooth-checkout-buildkite-plugin/releases/tag/v4.0.3)
- [v4.0.0+github.ssh.update](https://github.com/hasura/smooth-checkout-buildkite-plugin/releases/tag/v4.0.0+github.ssh.update)

I am hoping at-least one of them might help solve your problem. Please try them out.

**Comment by Gokulnath-Venugopal** [comment]
Thanks. I have created a workaround for the plugin and its working now. I will check and will let you know soon.

**Comment by imperfect-fourth** [comment]
this should be fixed in `v4.3.0`.
